src/interpreter.c

```diff
@@ -44,9 +44,9 @@ extern void JL_GC_ENABLEFRAME(interpreter_state*) JL_NOTSAFEPOINT;
 // This is necessary, because otherwise the analyzer considers this undefined
 // behavior and terminates the exploration
 #define JL_GC_PUSHFRAME(frame,n)     \
-  JL_CPPALLOCA(frame, sizeof(*frame)+((n) * sizeof(jl_value_t*)));                  \
+  JL_CPPALLOCA(frame, sizeof(*frame)+((n+3) * sizeof(jl_value_t*)));                  \
```

Comment:
This should be a separate PR, and can land immediately. Since it's just for clang-sa, probably doesn't need to be backported?
```suggestion
  JL_CPPALLOCA(frame, sizeof(*frame)+(((n)+1) * sizeof(jl_value_t*)));               \
  ((void**)&frame[1])[0] = NULL;                                                    \
  memset(&((void**)&frame[1])[1], 0, (n)*sizeof(jl_value_t*));                      \
  _JL_GC_PUSHARGS(&((void**)&frame[1])[1], n);
```